broad-fix-upload/metadata_fix.py

```diff
@@ -473,12 +474,15 @@ def main(argv=None):
              help="File(s) containing DONOR IDs to be excluded, use filename pattern to specify the file(s)", required=False)
     parser.add_argument("-i", "--include_donor_id_lists", dest="include_donor_id_lists", 
              help="File(s) containing DONOR IDs to be excluded, use filename pattern to specify the file(s)", required=False)
+    parser.add_argument("-s", "--is the workflow results complete", dest="workflow_result_status", default=True, type=bool,
```

Comment:
this argument could be named more informative, something like: `allow_partial_result_set`